adaptive_navigation/example/lib/default_scaffold.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:adaptive_navigation/scaffold.dart';
```

Comment:
Seems like almost this whole file is the same as `custom_scaffold.dart`, maybe combine them with an option to pass in a custom navigation resolver?

Reply:
I was thinking that because it is example code, it would be better not try to re-use code, so that when someone is looking to it as an example, it is all in one place. What do you think?

Reply:
gotcha, yeah I think that's fine then